### PR TITLE
Removed Cassandra waiting from upstart config.

### PR DESCRIPTION
**Pull request by jtk54**
Removed Cassandra waiting from upstart config. This hangs for people not using Cassandra on localhost. Replaced with reporting. @lwander PTAL. @ewiseblatt FYI.

### CODE REVIEW

File: etc/init/spinnaker.conf [hunk 1]
```diff
@@ -4,12 +4,12 @@ start on filesystem or runlevel [2345]
...
-    echo "Waiting for cassandra"
-    while ! nc -z localhost 9042; do
-      sleep 1
-    done
-    echo "Cassandra is ready"
+  if nc -z localhost 9042; then
+    echo "Cassandra server is ready on localhost."
+  else
+    echo "Cassandra does not appear to be running on localhost.
+If Spinnaker is configured to use Cassandra, start it and restart Spinnaker.
```
Comment (lwander) [inline etc/init/spinnaker.conf]: I'm not familiar with .conf format, but is this how comments work?

**Comment by duftler** [comment]
The original looping/blocking behavior makes me think that cassandra will sometimes (usually?) not be ready the first time it is pinged. Isn't there a risk that installations using cassandra will see the warning message almost every time this is executed?

File: etc/init/spinnaker.conf (hunk 1 above)
Comment (jtk54) [inline etc/init/spinnaker.conf]: It's a multi-line string to keep line length < 80.

**Comment by jtk54** [comment]
Discussed this out of band, pushed new changes.

File: InstallSpinnaker.sh [hunk 2]
```diff
@@ -678,6 +678,9 @@ fi
 ## Spinnaker
 install_spinnaker
 
+# touch a file to tell other scripts we installed Cassandra
```
Comment (duftler) [inline InstallSpinnaker.sh]: Please capitalize the comment and terminate it with a period.

File: pylib/spinnaker/change_cassandra.py [hunk 3]
```diff
@@ -104,12 +104,19 @@ def __init__(self, options):
...
+      open('/opt/spinnaker/cassandra/SPINNAKER_DISABLED_CASSANDRA', 'w').close()
     with open('/etc/init/cassandra.override', 'w') as f:
         f.write('manual')
     print 'Stopping cassandra service...'
     os.system('service cassandra stop')
 
   def enable_cassandra(self):
+    if os.path.exists('/opt/spinnaker/cassandra/SPINNAKER_DISABLED_CASSANDRA'):
+      os.remove('/opt/spinnaker/cassandra/SPINNAKER_DISABLED_CASSANDRA')
+      (open('/opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA', 'w')
```
Comment (duftler) [inline pylib/spinnaker/change_cassandra.py]: What's the reason this line has parens around it and the similar line in `disable_cassandra()` does not?

File: pylib/spinnaker/change_cassandra.py [hunk 4]
```diff
@@ -104,12 +104,19 @@ def __init__(self, options):
     self.__bindings.import_dict(config)
 
   def disable_cassandra(self):
+    if os.path.exists('/opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA'):
```
Comment (duftler) [inline pylib/spinnaker/change_cassandra.py]: Could these 2 paths be constants instead?

File: pylib/spinnaker/change_cassandra.py (hunk 3 above)
Comment (jtk54) [inline pylib/spinnaker/change_cassandra.py]: This makes each line < 80 chars.

**Comment by duftler** [comment]
LGTM once comments are addressed.
@lwander PTAL as well.

**Comment by lwander** [comment]
LGTM

**Pull request merged**